drivers/hmis/app/graphql/mutations/split_household.rb

```diff
@@ -0,0 +1,69 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Mutations
+  class SplitHousehold < BaseMutation
+    argument :splitting_enrollment_inputs, [Types::HmisSchema::EnrollmentRelationshipInput], required: true
+
+    field :new_household, Types::HmisSchema::Household, null: false
+    field :remaining_household, Types::HmisSchema::Household, null: false
+
+    def resolve(splitting_enrollment_inputs:)
+      map_enrollment_id_to_relationship = splitting_enrollment_inputs.map { |e| [e.enrollment_id, e.relationship_to_hoh] }.to_h
+
+      splitting_enrollment_ids = map_enrollment_id_to_relationship.keys
+      splitting_enrollments = Hmis::Hud::Enrollment.
+        where(id: splitting_enrollment_ids).
+        viewable_by(current_user).
+        includes(:household, :project)
+      access_denied! unless splitting_enrollments.size == splitting_enrollment_inputs.size
+
+      project = splitting_enrollments.map(&:project).uniq.sole
+      access_denied! unless current_permission?(permission: :can_split_households, entity: project)
+
+      donor_household = splitting_enrollments.map(&:household).uniq.sole
+      remaining_enrollments = donor_household.enrollments.where.not(id: splitting_enrollment_ids)
+
+      raise 'Splitting all clients to a new household is invalid' if remaining_enrollments.empty?
+      raise 'This operation would leave behind a household with no HoH, which is not allowed' unless remaining_enrollments.exists?(relationship_to_hoh: 1)
```

Comment:
Micro optimization: check against the array rather than making another db query with `exists?`:
```suggestion
      raise 'This operation would leave behind a household with no HoH, which is not allowed' unless remaining_enrollments.any?(&:head_of_household?)
```